docs/source/conf.py

```diff
@@ -48,6 +48,8 @@
     from numpyro import __version__  # noqaE402
     version = __version__
 
+    html_context = {'github_version': 'master'}
```

Comment:
On readthedocs, `github_version` always points to the corresponding tag (e.g. `master`, `0.4.1`, `v0.2.0`). However, when building locally, `github_version` will not be defined. This makes sure that colab link is valid even for local development. This is not important though.